homeassistant/components/homekit/type_switches.py

```diff
@@ -86,7 +86,7 @@ def __init__(self, *args):
     def is_activate(self, state):
         """Check if entity is activate only."""
         can_cancel = state.attributes.get(ATTR_CAN_CANCEL)
-        if self._domain == 'script' and not can_cancel:
+        if self._domain in ['script', 'scene'] and not can_cancel:
```

Comment:
`scenes` don't have the `can_cancel` attribute. So although this would technical work, I would like it better to be a separate if statement below the first (to just check if the domain is `scene`).

Reply:
I did it so it can be in one line but will change it